osquery/tables/system/darwin/homebrew_packages.cpp

```diff
@@ -56,10 +56,57 @@ std::vector<std::string> getHomebrewVersionsFromInfoPlistPath(
   return results;
 }
 
+Status getHomebrewPrefix(fs::path& prefix) {
+  // The Homebrew wrapper script finds the Library directory by taking the
+  // directory that it is located in and concatenating `/../Library`:
+  //   BREW_FILE_DIRECTORY=$(chdir "${0%/*}" && pwd -P)
+  //   export HOMEBREW_BREW_FILE="$BREW_FILE_DIRECTORY/${0##*/}"
+  // Note that the `-P` flag to pwd resolves all symlinks.
+  //
+  // Next, it will use given filename to find the prefix:
+  //   HOMEBREW_PREFIX = Pathname.new(HOMEBREW_BREW_FILE).dirname.parent
+
+  if (!pathExists(kHomebrewBinary).ok()) {
+    return Status(1, "No Homebrew binary found");
```

Comment:
nitpick, if you're not echoing or logging the status string/message then just return a `Status(1)` to not inflate the binary with unused const strings.